experimental/dds/tree/src/test/utilities/TestUtilities.ts

```diff
@@ -499,7 +499,7 @@ export const versionComparator = (versionA: string, versionB: string): number =>
 
 	assert(
 		versionASplit.length === versionBSplit.length && versionASplit.length === 3,
-		'Version numbers should follow semantic versioning.'
+		0x668 /* Version numbers should follow semantic versioning. */
```

Comment:
more asserts here which should probably be from "assert"

Reply:
do that and it won't build anymore, unless I remove the messages.